spec.html

```diff
@@ -42836,6 +43231,32 @@ <h1>
           1. Return *true*.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-hostgrowsharedarraybuffer" type="host-defined abstract operation">
+        <h1>
+          HostGrowSharedArrayBuffer (
+            _buffer_: a SharedArrayBuffer,
+            _newByteLength_: a non-negative integer,
+          ): either a normal completion containing either ~handled~ or ~unhandled~, or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It gives the host an opportunity to perform implementation-defined growing of _buffer_. If the host chooses not to handle resizing of _buffer_, it may return ~unhandled~ for the default behaviour.</dd>
+        </dl>
+        <p>The implementation of HostGrowSharedArrayBuffer must conform to the following requirements:</p>
+        <ul>
+          <li>The abstract operation must return either NormalCompletion(~handled~), NormalCompletion(~unhandled~), or an throw completion.</li>
+          <li>If the abstract operation does not complete normally with ~unhandled~, and _newByteLength_ &lt; the current byte length of the _buffer_ or _newByteLength_ > _buffer_.[[ArrayBufferMaxByteLength]], throw a *RangeError* exception.</li>
+          <li>Let _isLittleEndian_ be the value of the [[LittleEndian]] field of the surrounding agent's Agent Record. If the abstract operation completes normally with ~handled~, a WriteSharedMemory or ReadModifyWriteSharedMemory event whose [[Order]] is ~SeqCst~, [[Payload]] is NumericToRawBytes(~BigUint64~, _newByteLength_, _isLittleEndian_), [[Block]] is _buffer_.[[ArrayBufferByteLengthData]], [[ByteIndex]] is 0, and [[ElementSize]] is 8 is added to the surrounding agent's candidate execution such that racing calls to `SharedArrayBuffer.prototype.grow` are not "lost", i.e. silently do nothing.</li>
+        </ul>
+
+        <p>The default implementation of HostGrowSharedArrayBuffer is to return ~unhandled~.</p>
+
+        <emu-note>
```

Comment:
Can you move this note up above the preceding paragraph so that "the second requirement above" is more obvious?

Reply:
Done.